src/edu/stanford/nlp/pipeline/demo/corenlp

```diff
@@ -56,15 +70,21 @@ do_restart()
 {
   do_stop
   sleep 3
+  if [ -e "$CORENLP_DIR/corenlp.shutdown" ]; then
+    echo "Graceful stop failed. Force deleting the file:"
+    echo "$CORENLP_DIR/corenlp.shutdown"
+    rm -f "$CORENLP_DIR/corenlp.shutdown"
+  fi
   do_start
 }
 
 case $1 in
 start) do_start
 ;;
+status) do_status
+;;
 stop) do_stop
 ;;
 restart) do_restart
 ;;
 esac
-
```

Comment:
better restore the EOL (newline) here, see https://unix.stackexchange.com/a/18789

Reply:
fixed